up-l1/mqtt.adoc

```diff
@@ -35,8 +35,126 @@ MQTT is an OASIS standard messaging protocol for the Internet of Things (IoT). I
 
 For more information, please refer to https://mqtt.org/
 
-
 == Specifications
 
+=== UAttribute Mapping
+
+MQTT 5 supports custom header fields, and we leverage this to map the UAttribute values into the MQTT header.
```

Comment:
if this specification covers MQTT 5 only, then IMHO the document should be renamed to `mqtt5.adoc` and this fact should be made explicit in the Overview section already.